web-client/extras/lib/StringUtils.ts

```diff
@@ -0,0 +1,29 @@
+export class StringUtils {
+    static isMultibyte(str: string): boolean {
```

Comment:
`isMultibyte` seems like the wrong name. It also seems to be used as a `isAscii` check, which this function does not implement.

Reply:
It checks if all characters in the string are in the ascii range. So it's definitely not multibyte when this returns true. Or what am I missing?

Reply:
As far as I can tell, it checks whether the string has any [surrogates](https://en.wikipedia.org/w/index.php?title=UTF-16&oldid=1245839466#U+D800_to_U+DFFF_(surrogates)) in UTF-16 encoding. ASCII would be checking that all characters are between U+0 and U+7F.

Reply:
Hmm. So what should this method do instead? Or should I research?

Reply:
I don't really know myself. Looking at the usage, it looks like [isWellFormed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/isWellFormed) would be a good replacement. It checks whether the JS string is actually a well-formed Unicode string. This guarantees that it can be encoded as UTF-8. Looking at the MDN page, it might be a new API that we can't rely on yet, so maybe there's a polyfill version?

Reply:
Done in https://github.com/nimiq/core-rs-albatross/pull/2715/commits/c805864e562615813f27b57ee43d895f867bdb93